mappings/net/minecraft/client/gui/menu/GuiSettingsVideo.mapping

```diff
@@ -2,6 +2,8 @@ CLASS none/bgv net/minecraft/client/gui/menu/GuiSettingsVideo
 	FIELD a title Ljava/lang/String;
 	FIELD f parent Lnone/bgn;
 	FIELD g settings Lnone/bdt;
+	FIELD h widgetList Lnone/bex;
```

Comment:
Don't GUIs already have a collection of widgets? Is this special in some way? If so the name should reflect that.